src/components/Personalization/createViewCacheManager.js

```diff
@@ -49,9 +49,13 @@ export default ({ createProposition }) => {
     const updateCacheDeferred = defer();
 
     cacheUpdateCreatedAtLeastOnce = true;
-    viewStoragePromise = viewStoragePromise.then(oldViewStorage =>
-      updateCacheDeferred.promise.catch(() => oldViewStorage)
-    );
+    viewStoragePromise = viewStoragePromise.then(oldViewStorage => {
```

Comment:
Can we please add an inline comment describing this new behavior and remember to add this config to the docs?
Thanks!